jbmc/src/java_bytecode/java_string_library_preprocess.cpp

```diff
@@ -1548,6 +1548,21 @@ void java_string_library_preprocesst::initialize_conversion_table()
   cprover_equivalent_to_java_string_returning_function
     ["java::org.cprover.CProverString.substring:(Ljava/Lang/"
      "StringBuffer;II)Ljava/lang/String;"] = ID_cprover_string_substring_func;
+  cprover_equivalent_to_java_string_returning_function
+    ["java::org.cprover.CProverString.toString:(Z)Ljava/lang/String;"] =
+      ID_cprover_string_of_bool_func;
+  cprover_equivalent_to_java_string_returning_function
+    ["java::org.cprover.CProverString.toString:(C)Ljava/lang/String;"] =
+      ID_cprover_string_of_char_func;
```

Comment:
:lollipop:

Reply:
We could use
```
char arr[] = {c};
return new String(arr);
```
which calls `ofCharArray()` which creates a `StringBuilder` and uses `append()` to insert the char and then returns `StringBuilder.toString()`. Any idea if this might have performance implications?